OctopusDSC/OctopusDSC.psd1

```diff
@@ -61,7 +61,7 @@ CLRVersion = '4.0'
 # NestedModules = @()
 
 # Functions to export from this module, for best performance, do not use wildcards and do not delete the entry, use an empty array if there are no functions to export.
-FunctionsToExport = 'Get-TargetResource', 'Set-TargetResource', 'Test-TargetResource'
+FunctionsToExport = 'Get-TargetResource', 'Set-TargetResource', 'Test-TargetResource', 'Sing-Out'

```

Comment:
Whats this `Sing-Out`?

Reply:
Gah. that needs to go. That was originally just to confirm the external file was loading properly.